src/meta/meta_ckpt.c

```diff
@@ -630,7 +630,8 @@ __wt_metadata_set_base_write_gen(WT_SESSION_IMPL *session, uint64_t max_write_ge
      * We track the maximum page generation we've ever seen, and I'm not interested in debugging
      * off-by-ones.
      */
-    S2C(session)->base_write_gen = WT_MAX(ckpt.write_gen, max_write_gen) + 1;
+    S2C(session)
+      ->max_write_gen = S2C(session)->base_write_gen = WT_MAX(ckpt.write_gen, max_write_gen) + 1;
```

Comment:
It is worth setting `conn` for this function to avoid the undesirable line break here. Also please set them in alphabetical order.